lib/matplotlib/backends/backend_svg.py

```diff
@@ -68,6 +68,12 @@
 # OF THIS SOFTWARE.
 # --------------------------------------------------------------------
 
+def _sorted_iteritems(a):
```

Comment:
Can you move these into `matplotlib.cbook` so we don't have multiple definitions of them floating around?